### PR TITLE
Learn - remove tag attribute form front matter, part 1

### PR DESCRIPTION
**Pull request by OnkarRuikar**

### CODE REVIEW

File: files/en-us/learn/accessibility/what_is_accessibility/index.md [hunk 1]
```diff
@@ -1,22 +1,6 @@
 ---
 title: What is accessibility?
```
Comment (hamishwillee) [inline files/en-us/learn/accessibility/what_is_accessibility/index.md:2]: FYI, some cases have this in " and others do not.

File: files/en-us/learn/getting_started_with_the_web/what_will_your_website_look_like/index.md [hunk 2]
```diff
@@ -1,18 +1,8 @@
...
-  - Composing
-  - Content
-  - Deprecated
-  - Design
-  - Fonts
-  - Learn
-  - Simple
-  - "l10n:priority"
-  - step by step
+status:
```
Comment (hamishwillee) [inline files/en-us/learn/getting_started_with_the_web/what_will_your_website_look_like/index.md:4]: If you're scripting this, it works :-)

**Review by hamishwillee — APPROVED** [review]
Excellent. FYI only, seems odd to see pages without page types.

**Comment by teoli2003** [comment]
For bookkeeping purposes: mdn/mdn#262

**Pull request merged**